docs/modules/ROOT/pages/how-tos/upgrade-6.x-to-7.x.adoc

```diff
@@ -0,0 +1,17 @@
+= Upgrade from v6 to v7
+
+This guide describes the steps to perform an upgrade of the component from version v6 to v7.
+
+== Changes
+
+Keycloak will be upgraded from v15.0.2 to v16.1.1 and if the built-in PostgreSQL database is used, PostgreSQL is going to be updated from 11.8.0-debian-10-r61 to 11.11.0-debian-10-r31.
+
+== Step-by-step guide
+
+When upgrading the component, the following actions are required if the built-in database is used:
+
+. Take a backup of the built-in database!
+. Unchangable attributes in the postgres statefulset are going to be changed.
+  If ArgoCD can't apply the statefulset, it needs to be deleted:
+  `kubectl -n syn-keycloak delete sts keycloak-postgresql`
+  This don't touch the PVC `data-keycloak-postgresql-0`.
```

Comment:
(I'm not quite sure if that renders nicely in the docs preview)
```suggestion
When upgrading the component, the following actions are required if the built-in database is used:

. Make a backup of the built-in database.

. Apply the parameter changes.

. Compile and push the cluster catalog.

. If ArgoCD can't apply the updated StatefulSet, it needs to be deleted: `kubectl -n syn-keycloak delete sts keycloak-postgresql`.
  +
  [NOTE]
  ====
  This step is necessary since the upgrade changes immutable properties in the Postgres StatefulSet if using the built-in database.
  This won't delete the PVC `data-keycloak-postgresql-0`.  
  ====

. Verify that ArgoCD can sync all resources.
```